lib/http-credentials.js

```diff
@@ -0,0 +1,54 @@
+/*!
+ * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const {asyncHandler} = require('bedrock-express');
+const bedrock = require('bedrock');
+
+const brJsonldDocumentLoader = require('bedrock-jsonld-document-loader');
+const vc = require('vc-js');
+const {Ed25519KeyPair} = require('crypto-ld');
+const {suites: {Ed25519Signature2018}} = require('jsonld-signatures');
+
+const mockSigningKey = {
+  type: 'Ed25519VerificationKey2018',
+  // eslint-disable-next-line max-len
+  privateKeyBase58: '4m1ybR6T3RSWdD31SUTPpi9THjiMwUhwqEP5W7c5WDgaz5vdWoAirhUkRJUrQYfwHoLFeGWV4RAYHbWBwLfC2qrm',
+  publicKeyBase58: '2gui9QRXHkkyiCukDRa6SKEc856CdsxUU2CeL6JR32ER'
+};
+
+bedrock.events.on('bedrock-express.configure.routes', app => {
+  //const cfg = config['bedrock-vc-issuer'];
+  const cfg = {
+    routes: {
+      issueCredential: '/credentials/issueCredential',
+    }
+  };
+  const {routes} = cfg;
+
+  app.post(
+    routes.issueCredential,
+    // TODO: add validation
+    asyncHandler(async (req, res) => {
+      const {
+        credential,
+        // FIXME: what are options used for?
+        // options,
```

Comment:
See: https://w3c-ccg.github.io/vc-issuer-http-api/index.html#/internal/issueCredential

Options are arguments passed to the issuance process that are used to modify the signed output. For example, `assertionMethod` is an option that specifies which key the requestor would like to be used.